bentoml/_internal/runner/container.py

```diff
@@ -178,6 +178,70 @@ def from_batch_payloads(
         return cls.batches_to_batch(batches, batch_dim)
 
 
+class DMatrixContainer(
```

Comment:
when we implement this container, could we move it to the xgboost module? we are doing that for the PyTorch and TF tensor, so that those containers are only registered when those modules got imported by user code

Reply:
Actually I only included this because it was a pain to separate out properly but I ended up making a commit for it anyway so we can just drop it.

DMatrix has no native way to concatenate, so if we wanted to do that we'd either have to hack memory or save to binary -> use their internal format to hack together the files -> load.